content/deploy/community-cloud/manage-your-app/edit-your-app.md

```diff
@@ -15,72 +15,69 @@ Spin up a cloud-based development environment for your deployed app in minutes.
 
-   ![Create your GitHub Codespace](/images/streamlit-community-cloud/deploy-codespaces-2.png)
+   ![Your new GitHub Codespace](/images/streamlit-community-cloud/deploy-template-blank-codespace.png)
 
-4. Wait for GitHub to set up your codespace.
+1. (Optional) Open the app preview in another tab for more room to work.
 
-   ![GitHub preparing your codespace](/images/streamlit-community-cloud/deploy-codespaces-3.png)
+   If you have multiple monitors and would like a little more room to work, you can open your app preview in another tab instead of using the Simple Browser with Visual Studio code. Just copy the URL from the Simple Browser into another tab, then close the Simple Browser. Now you have more room to edit your code. This page will continue to refer to your code on the left and your preview on the right.
 
-5. GitHub will automatically execute the commands to launch your Streamlit app within your codespace. Your app will be visible in a "Simple Browser" on the right. This may take a minute to complete from when your codespace first appears on screen.
+1. Make a change to your app.
 
-   ![Your new GitHub Codespace](/images/streamlit-community-cloud/deploy-sample-codespace.png)
+   When you make changes to your app, the file is automatically saved within your codespace. Your edits do not affect your repository or deployed app until you commit those changes. We will describe committing your changes in a later step. The app preview shown on the right is local to your codespace.
 
-6. When you make changes to your app, the file is automatically saved within your codespace. Your edits do not affect your repository unless you choose to commit those changes. We will describe committing your changes in a later step.
+1. In order to see updates automatically reflected on the right, click "**Always rerun**" when prompted after your first edit.
 
-   In order to see updates automatically reflected on the right, click "**Always rerun**" when prompted after an edit.
+   ![Select "Always rerun" to automatically see edits in your running app](/images/streamlit-community-cloud/deploy-template-blank-codespace-edit.png)
 
-   ![Edit the title of your sample Streamlit app](/images/streamlit-community-cloud/deploy-sample-edit-title.png)
-   ![Select "Always rerun" to automatically see edits in your running app](/images/streamlit-community-cloud/deploy-sample-edit-rerun.png)
+   Alternatively, you can click "**Rerun**" to avoid unecessary reruns while writing code. Since your code will continually resave, automatically rerunning the app will raises errors when you pause partway through a line of code. Regardless which you choose, you can change the setting through the app chrome. Just click the the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) in the upper-right corner of your preview app, click "**Settings**," and toggle "**Run on save**."
```

Comment:
```suggestion
   Alternatively, you can click "**Rerun**" to avoid unnecessary reruns while writing code. Because your code is continually saved, automatically rerunning the app will raises errors when you pause partway through a line of code. Regardless of which you choose, you can change the setting through the app chrome. Just click the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) in the upper-right corner of your preview app, click "**Settings**," and then toggle "**Run on save**."
```